response.cc

```diff
@@ -61,3 +104,110 @@ std::string Response::ToString() const
 
 	return ss.str();
 }
+
+Response::ResponseCode Response::IntToResponseCode(int code) {
+	switch(code) {
```

Comment:
Aren't these already equivalent because of the enum?

(I'm planning on replacing this and the other switch later)